checking-account.rb

```diff
@@ -0,0 +1,35 @@
+module Bank
+  class CheckingAccount < Account
+    attr_reader :used_checks
+
+    def initialize(ident, open_date, balance)
+      super
+      @used_checks = 0
+    end
+
+    def withdraw(withdraw_amount)
+      super(withdraw_amount + 100)
```

Comment:
Yes! Awesome use of the original account method to take care of this functionality for you